### ISSUE TITLE
Extention not available on Chrome Store

### ISSUE THREAD

**Issue by marek90** [T3aHat/DeepLopener#15]
Hi, when I look up the extension on [Chrome Web Store](https://chrome.google.com/webstore/detail/almdndhiblbhbnoaakhgefcpmbaoljde) it's not available – 404. Is there a problem? (technical, license, etc...)

**Comment by T3aHat** [comment]
Hi, marek90!
Thank you for your interest in this extension.

We're so sorry but DeepLopener is no longer available on ChromeWebStore.
This is in response to a warning from deepl.com that users may misunderstand that it is official.
But, we will continue to release DeepLopener on GitHub and add new features as soon as we feel like it.

**Comment by marek90** [comment]
Hi @T3aHat, thank you for the prompt and straightforward answer! 

Wouldn't changing the name and/or logo solve the problem? The only reason I'm asking is that while you can still load up the extension manually, possible future updates won't get installed automatically (and so for less technical users, they would be left on older versions).

**Comment by T3aHat** [comment]
Yes, we believe that changing the name and logo will solve the problem.
However, we have no plans at this time to republish it in the store with the changes, as we anticipate that our busy schedule will make it difficult to continue to maintain it in the future.
